mgmt/api/CoreAPI.cc

```diff
@@ -880,6 +880,21 @@ EventSignalCbUnregister(const char *event_name, TSEventSignalFunc func)
   return cb_table_unregister(local_event_callbacks, event_name, func);
 }
 
+/*-------------------------------------------------------------------------
+ * HostStatusSet
+ *-------------------------------------------------------------------------
+ * Iterates through the RecordsConfig table, and for all stats
+ * (type PROCESS, NODE, CLUSTER), sets them back to their default value
+ * If one stat fails to be set correctly, then continues onto next one,
+ * but will return TS_ERR_FAIL. Only returns TS_ERR_OKAY if all
+ * stats are set back to defaults successfully.
+ */
+TSMgmtError
+HostStatusSet(const char *name, bool value)
```

Comment:
It's not obvious what `HostStatusSet("foo.com", true)` would do. If it was `HostStatusSet("foo.com", HOST_STATUS_UP)` that would be a lot more obvious.